ssl/ssl_local.h

```diff
@@ -1586,6 +1587,7 @@ struct ssl_connection_st {
             /* OCSP response received or to be sent */
             unsigned char *resp;
             size_t resp_len;
```

Comment:
Are you really need to keep this pair of variables, now that you are storing all the OSCP_RESPONSE in a stack ?

Reply:
Yes, it is absolutely needed to properly keep the semantics for the single response set/get ctrls.